src/mc_tools.jl

```diff
@@ -197,30 +175,24 @@ of the three `_solve` methods to use.
 distribution of `mc.p`
 
 """
-function mc_compute_stationary{T}(mc::MarkovChain{T}; method::Symbol=:gth)
-    @compat solvers = Dict(:gth => gth_solve,
-                           :lu => lu_solve,
-                           :eigen => eigen_solve)
+function mc_compute_stationary{T}(mc::MarkovChain{T}; method=:gth)
+    solvers = Dict(:gth => gth_solve, :lu => lu_solve, :eigen => eigen_solve)
```

Comment:
We should also make sure that the `method` argument is typed as `::Symbol`. 

(we should restore exactly what was here originally -- it reverts both changes)